doc/api/test.md

```diff
@@ -99,6 +99,17 @@ test('callback failing test', (t, done) => {
     done(new Error('callback failure'));
   });
 });
+
+function failFunction() {
+  throw new Error('This error will be caught');
+}
+
+// It's the way to catch expected errors i your test
+test('test catch error', (t) => {
+  assert.throws(() => failFunction(), {
+    message: 'This error will be caught',
+  });
+});
```

Comment:
Let's remove this one that feels out of place. Instead, what we can do is modify `synchronous passing test` and `asynchronous passing test` to include use of `assert.throws()` and `await assert.rejects()`.
Also, we advice against using `error.message` to prefer `error.code`, let's showcase that in our docs.
```suggestion
```